Mathlib/Data/Set/Finite.lean

```diff
@@ -789,21 +789,21 @@ theorem Finite.unionₛ {s : Set (Set α)} (hs : s.Finite) (H : ∀ t ∈ s, Set
   apply to_finite
 #align set.finite.sUnion Set.Finite.unionₛ
 
-theorem Finite.bUnion {ι} {s : Set ι} (hs : s.Finite) {t : ι → Set α} (ht : ∀ i ∈ s, (t i).Finite) :
+theorem Finite.bunionᵢ {ι} {s : Set ι} (hs : s.Finite) {t : ι → Set α} (ht : ∀ i ∈ s, (t i).Finite) :
     (⋃ i ∈ s, t i).Finite := by
   classical
     cases hs
-    haveI := fintype_bUnion s t fun i hi => (ht i hi).Fintype
+    haveI := fintypeBUnionᵢ s t fun i hi => (ht i hi).fintype
     apply to_finite
-#align set.finite.bUnion Set.Finite.bUnion
+#align set.finite.bUnion Set.Finite.bunionᵢ
 
-/-- Dependent version of `finite.bUnion`. -/
-theorem Finite.bUnion' {ι} {s : Set ι} (hs : s.Finite) {t : ∀ i ∈ s, Set α}
+/-- Dependent version of `finite.bunionᵢ`. -/
```

Comment:
```suggestion
/-- Dependent version of `Finite.bunionᵢ`. -/
```
Even better: we're preparing a python script that will make these fixes for you. So I suggest that you focus on the Lean errors/name changes, and ignore comments and docstrings for now.

Reply:
oof I should've read this before I did it all :(